### PR TITLE
Handle subjects page translation

### PR DESCRIPTION
**Pull request by RoyEJohnson**

### CODE REVIEW

File: src/app/pages/subjects/new/specific/book-viewer.js [hunk 1]
```diff
@@ -7,17 +7,44 @@ import {faCaretDown} from '@fortawesome/free-solid-svg-icons/faCaretDown';
...
+    'es': {
+        orderPrint: 'Solicite una copia impresa',
+        getBook: 'Consigue el libro',
+        viewOnline: 'Ver en línea',
+        downloadPdf: 'Descargar un PDF',
+        instructorResources: 'Recursos del instructor',
+        studentResources: 'Recursos para estudiantes'
+    }
+};
+
```
Comment (nathanstitt) [inline src/app/pages/subjects/new/specific/book-viewer.js:32]: have we looked into using a proper i18n framework? maybe pickup whatever REX uses?

File: test/setupFile.js [hunk 2]
```diff
@@ -18,8 +18,6 @@ global.document.createRange = () => ({
 });
 global.document.getSelection = () => {};
 
-window.MutationObserver = function () {};
-window.MutationObserver.prototype.observe = function () {};
 window.HTMLElement.prototype.scrollIntoView = jest.fn();
```
Comment (nathanstitt) [inline test/setupFile.js:21]: 😬 I'm guessing jsdom now has MutationObserver ?

**Review by nathanstitt — APPROVED** [review]

File: test/setupFile.js (hunk 2 above)
Comment (RoyEJohnson) [reply test/setupFile.js:21]: I realized I was no longer using MutationObserver in my code, so i didn't need to mock it.

File: src/app/pages/subjects/new/specific/book-viewer.js (hunk 1 above)
Comment (RoyEJohnson) [reply src/app/pages/subjects/new/specific/book-viewer.js:32]: This is a good idea.

**Pull request merged**